index.html

```diff
@@ -15,276 +15,291 @@
-                <img class="img-sec4" src="images/Snapshoot Portfolio section 3.png" alt="project image">
+            <p class="section-description1">A smart assistant to make driving more safe, efficient, and fun by
+                unlocking
+                your most expensive computer: your car.</p>
+            <ul class="tags">
+                <li class="li11">
+                    <article class="text1">html</article>
+                </li>
+                <li class="li222">
+                    <article class="text22"> Ruby on rails </article>
+                </li>
+                <li class="li22">
+                    <article class="text2">css</article>
+                </li>
+                <li class="li33">
+                    <article class="text3">javascript
+                </li>
+            </ul>
+            <div class="action">
+                <button class="btn btn1" id="pop1">See Project</button>
             </div>
-            <div class="left-block00">
-                <div class="experience1">
-                    <h1 class="section-title1">Multi-Post
-                        Stories</h1>
-                    <ul class="list">
-                        <li class="list1">CANOPY</li>
-                        <li class="list2">Back End Dev</li>
-                        <li class="list3">2015</li>
-                    </ul>
-                </div>
-                <p class="section-description0">A daily selection of privately personalized reads; no
-                    accounts or sign-ups
-                    required.</p>
-                <p class="section-description1">Experimental content creation feature that allows users to add to an
-                    existing story over the course of a day without spamming their friends.</p>
-                <ul class="tags">
-                    <li class="li11">
-                        <article class="text1">html</article>
-                    </li>
-                    <li class="li222">
-                        <article class="text22"> Ruby on rails </article>
-                    </li>
-                    <li class="li22">
-                        <article class="text2">css</article>
-                    </li>
-                    <li class="li33">
-                        <article class="text3">javascript
-                    </li>
+        </div>
+
+    </section>
+    <section class="sections">
+
+    </section>
+    <section class="section3">
+        <div class="right-block">
+            <img class="image-section3" src="images/Snapshoot Portfolio2.png" alt="project image">
+            <img class="img-sec3" src="images/Snapshoot Portfolio section33.png" alt="project image">
+        </div>
+        <div class="left-block">
+            <div class="experience">
+                <h1 class="section-title">Tonic</h1>
+                <h1 class="section-titlef">Tonic</h1>
+                <ul class="list">
```

Comment:
- [ ] It would be better if our modal fits in as we have in the original design, however, currently, it is hard to close the modal or even read contents of the modal. Let us kindly fix it
![Screenshot from 2023-01-06 22-39-34](https://user-images.githubusercontent.com/46388113/211111331-af6a7ac1-9f4a-4b3c-8e8b-21cdc257cce4.png)